src/envoy/http/jwt_auth/pubkey_cache.h

```diff
@@ -46,6 +48,18 @@ class PubkeyCacheItem {
     for (const auto& aud : jwt_config_.audiences()) {
       audiences_.insert(SanitizeAudience(aud));
     }
+
+    if (jwt_config_.has_local_jwks() &&
+        jwt_config_.local_jwks().specifier_case() ==
+            ::envoy::api::v2::core::DataSource::kInlineString) {
+      Status status =
+          SetKey(Base64::decode(jwt_config_.local_jwks().inline_string()));
```

Comment:
So do you expect the key is base64 encoded?

Reply:
does it mean pilot side needs encoding the key ?

Reply:
Yes,  Envoy is doing a lot of JsonConvert for filter config.   We could not just put json as value here

Reply:
Please don't do this, if you want inline them with base64 in YAML/JSON, just use `inline_bytes`, by proto/json spec they are base64 encoded. Don't mix them up.

Reply:
Done